spec/atm_spec.rb

```diff
@@ -66,6 +61,15 @@
       subject.do_transaction(5, account)
       expect(subject.funds).to eq 995
     end
+
+    it 'does not allow for withdraw of 105$ from ATM if money not avilable in person account' do
+      output = { status: false,
+                 message: :no_sufficient_balance,
+                 date: Date.today.strftime('%F')
+        }
+        #binding.pry
+      expect(subject.withdraw(105, account.pin, account.exp_date, account)).to eq output
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)